### DISCUSSION TITLE
How do I run the docs locally?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [fastapi/fastapi#5391] [Questions] [answered]
I've cloned the fastapi repo, but I'd like the generate and run the docs locally (offline). What is the command to doing this?

**Comment by uysalserkan** [comment]
Docs folder has generally markdown files, if you install an app or an extension on IDE/IDEA you could see docs markdown files.

**Reply by CodyBontecou** [reply]
Oh, definitely. That's what I fell back on, I was just hoping to find documentation on how to build the pretty docs locally.

**Comment by yezz123** [comment, marked as answer]
> I've cloned the fastapi repo, but I'd like the generate and run the docs locally (offline). What is the command to doing this?

Well after cloning you can try to install dependencies and then run this https://github.com/tiangolo/fastapi/blob/master/scripts/docs-live.sh

**Comment by Jedore** [comment]
I export docs by accessing `http://host:port/openapi.json` and then import docs into other tools like [Apifox](https://www.apifox.cn/help/app/import/) which can serve docs. Also, you can  [export](https://www.apifox.cn/help/app/export/#%E5%8A%9F%E8%83%BD%E8%AF%B4%E6%98%8E) docs to local file in `HTML` type  from `ApiFox`, then access the `HTML` file offline.
